session/handler.go

```diff
@@ -903,3 +908,91 @@ func RespondWitherrorGenericOnAuthenticated(h herodot.Writer, err error) httprou
 		h.WriteError(w, r, err)
 	}
 }
+
+// Exchange Session Token Parameters
+//
+// swagger:parameters exchangeSessionToken
+//
+//nolint:deadcode,unused
+//lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
+type exchangeSessionToken struct {
+	// The part of the code return when initializing the flow.
+	//
+	// required: true
+	// in: query
+	InitCode string `json:"init_code"`
+
+	// The part of the code returned by the return_to URL.
+	//
+	// required: true
+	// in: query
+	ReturnToCode string `json:"return_to_code"`
```

Comment:
In the future, this endpoint may support other token exchanges as well. Should we somehow prefix / clarify what code you're exchanging here?

Reply:
I'm not sure what other usages you think of here. If there are other types of tokens, wouldn't this be a different end point? It would probably be easier to reason about the security then?